policy/policy.sample.json

```diff
@@ -0,0 +1,71 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1",
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
+                {
+                    "Tenant":"tenant1", #optional if TenantID below is provided 
+                    "TenantID": "3A6E467F-370B-43EA-BCEC-C28912EDD752", # optional but good to have to relate things as names are not unique
+                    "Segment": "Segment1", # optional
+                    "SegmentID": "01CA06A0-429B-4EE1-9185-F17D5CE4D0F2",
+                },
+                {
+                    "Tenant":"tenant1",
+                    "Segment": "Segment2",
+                },
+            ],
+            "Tags": [
+                {
+                    "Key": "Platform",
+                    "Value": "Openstack",
+                },
+                {
+                    "Key": "TenantID",
+                    "Value": "D46FD18B-A381-4297-A6D4-9DA594EE71C8",
+                },
+            ], # meta data attached to policies for various external enviorments like openstack/kybernetes
+            "Direction" : "Egress", # can be Egress or Ingress.
+            "Rules": [
+                {
+                    "CidrBlock": "0.0.0.0/0", # IP from L3 header
```

Comment:
Organization.

What are the top element of a policy? We have to options here:

    1. AppliedTo, Peer, Direction, Rules

or

    2. AppliedTo, Direction, Rules, [CIDRBlock | cluster | tenant/segment-combo | whatever-else-we-come-up-with-in-the-future ]

My point is that the peer definition can change over time. More possibilities to define peers can be developed. This makes the top-level of the policy ... not orderly.

Someone may look at a new version of a policy definition (maybe in a new release from us). They find "foobar" at the top level. They have no idea at all that this might be a new means of defining a peer.

However, if it appears under a "peer" heading then it's immediately clear.